app/presenters/vendor_api/application_presenter/deferred_offer.rb

```diff
@@ -7,8 +7,15 @@ def schema
         offer: {
           status_before_deferral: application_choice.status_before_deferral,
           offer_deferred_at: application_choice.offer_deferred_at&.iso8601,
+          confirm_deferral_recruitment_cycle_year: confirm_deferral_recruitment_cycle_year,
```

Comment:
I find `confirm_deferral_recruitment_cycle_year` a little confusing, how about `deferred_to_recruitment_cycle_year` instead?

Reply:
I also don't find the name very intuitive - do we have a name for the 'make deferred offer able to be reconfirmed' event?